components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/builder/FileBasedConfigurationBuilder.java

```diff
@@ -666,6 +768,16 @@ public void setAuthenticationEndpointMissingClaimsURL(String authenticationEndpo
         this.authenticationEndpointMissingClaimsURL = authenticationEndpointMissingClaimsURL;
     }
 
+    public String getAuthenticationEndpointMissingClaimsURLV2() {
```

Comment:
shall we add method descriptions. check other methods a well

Reply:
Resolved with https://github.com/wso2/carbon-identity-framework/pull/6222/commits/9a194170f6bf003a497ee9a6a4ee4405682ceca2